### DISCUSSION TITLE
export doesn't work

### DISCUSSION THREAD

**Discussion by AlbertMinrich** [nilsteampassnet/TeamPass#3785] [Q&A] [answered]
I activated the export function. Now I'm trying to export data via "Write list of items to readable file". I can choose between PDF and CSV. The export file also seems to be created because the item "Download file" appears. However, if I click on it, "File is not available on the website" comes up.

What can I do?
Thanks
Albert

**Comment by nilsteampassnet** [comment]
Is the file created in folder files?
Check your apache log if the rights on that folder are correct

**Reply by AlbertMinrich** [reply]
If I click on the download button, in the apache log this line appears:
teampass.internal.lo:443 172.16.223.29 - - [13/Jul/2023:12:25:47 +0200] "GET /files/print_out_csv_1689243933_df41a3b51994f88.csv HTTP/1.1" 404 1096 "https://teampass.internal.lo/index.php?page=items" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"

But in the folder /var/www/teampass/files/ there is no file print_out_csv_1689243933_df41a3b51994f88.csv
there is no current file at all in the folder.
There is only one file print_out_csv_ file and about 20 with cryptic letters/numbers. These are all export files, at least they contain logins and passwords, but all files are from January 2019, so it seems to have worked at some point.

Regards
Albert

**Reply by AlbertMinrich** [reply, marked as answer]
OK, it's solved. In the settings > Files folder another path was set, deviating from the standard
I changed it back and it works.

Thanks and regards
Albert
